GSDemo/app/build.gradle

```diff
@@ -35,8 +33,6 @@ android {
                 arguments '-DWITH_SWIG=ON'
                 arguments '-DSWIG_WITH_JAVA=ON'
                 arguments '-DCMAKE_BUILD_TYPE=Release'
-                arguments '-DCMAKE_CXX_FLAGS=-llog'
-                arguments '-DCMAKE_C_FLAGS=-llog'
```

Comment:
While not explicitly necessary these arguments if I am not entirely misstaken allows debug prints from the C/C++ code (iso22133/isoObject) to be passed from the compiled .so files to logcat. I.e including  <android/log.h> in any c/c++ file that is beeing compiled and then using   __android_log_print(ANDROID_LOG_ERROR, "TRACKERS", "%s", Str); in the relevant code sections. 

I might be remembering this wrong though maybe it works anyway, will check sometime in the future. If this works anyway they could probably be removed.